src/content/self-hosted/on-premise.mdx

```diff
@@ -0,0 +1,195 @@
+
+- A subdomain to which you can add a wildcard DNS record
+- A Kubernetes cluster
+- A working installation of kubectl
+- A working installation of Helm v3
+- An OAuth application
+- An Okteto license (optional)
+
+### Subdomain
+
+You'll need access to a subdomain to which you can add a wildcard DNS record, such as dev.example.com.
+
+### Deploy a Kubernetes cluster
+
+Okteto supports Kubernetes versions 1.21 through 1.23.
+
+We recommend the following specs:
+- v1.23
+- A pool with at least 2 nodes
+- 250 GB per Standard SSD Managed Disk
+
+You'll be using the cluster's API server endpoint when configuring Okteto.
+
+### Installing kubectl
+
+Follow the official documentation for [installing kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/). Once installed, configure `kubectl` to [talk to your new cluster](https://www.civo.com/learn/kubernetes-cluster-administration-using-civo-cli#downloading-the-cluster-39-s-configuration).
+
+### Installing Helm v3
+
+Follow the official documentation for [installing the latest release of Helm v3](https://helm.sh/docs/intro/install/).
+
+### Creating the OAuth application
+
+> Okteto supports OAuth providers like Google, GitHub, or OpenID Connect to handle user authentication. For this guide, we'll focus on Okta with OpenID Connect. All the supported configuration settings [are described here](/docs/self-hosted/administration/configuration/#auth).
+
+Use this group of settings when using an OpenID Connect provider as your authentication provider.
+
+```yaml
+auth:
+  openid:
+    enabled: true
+    clientId: clientid
+    clientSecret: clientSecret
+    group: my-group
+    endpoints:
+      issuer: https://your-provider
+      authorization: https://your-provider/authorization
+    mapping:
+      externalIDKey: nickname
+      nameKey: name
+      emailKey: email
+      pictureKey: picture
+      groupsKey: groups
+```
+
+The `group` field is optional. Only members of the group will be allowed to log in into your Okteto instance. An empty `group` field permits any user to log in.
+
+The `issuer` and `authorization` endpoints must match the value returned in the provider config discovery.
+
+The `mapping` fields are optional. Use them to configure the mapping between Okteto's user attributes and the claim coming from your authentication provider.
+
+> Your provider needs to support the [UserInfo endpoint](https://openid.net/specs/openid-connect-core-1_0.html#UserInfo) in order to be used with Okteto. This authentication option follows the OpenID standard, and it has been validated with Okta, PingIdentity, and GitLab.
+
+## Adding the Okteto Helm repository
+
+You'll need to add the Okteto repository in order to be able to install Okteto:
+
+```console
+helm repo add okteto https://charts.okteto.com
+helm repo update
+```
+
+### Getting your Okteto License
+
+Okteto is free for small teams. You get all the features of [Okteto](/free-trial/) for up to 3 users with 3 namespaces each.
```

Comment:
The free trial is for 100 users. We no longer have the per-namespace limit.

Reply:
Free trial is not the same than Starter. starter is the one that's free for up to 3 users without a license.

Reply:
In the guides we should encourage the free trial

Reply:
@rberrelleza could you propose an alternative to this paragraph?
```
Okteto is free for small teams. You get all the features of [Okteto](/free-trial/) for up to 3 users with 3 namespaces each.
```